ui/components/app/edit-gas-fee-popover/network-statistics/index.scss

```diff
@@ -7,37 +7,32 @@
     height: 56px;
     display: flex;
     align-items: center;
-    justify-content: space-around;
-
-    &__separator {
-      border-left: 1px solid var(--ui-2);
-      height: 65%;
-    }
-
-    &__field {
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      justify-content: center;
-      width: 30%;
+    justify-content: center;
+  }
 
-      &-data {
-        color: var(--ui-4);
-        font-size: 12px;
-        text-align: center;
-      }
+  &__field {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex: 1;
 
-      &-label {
-        color: var(--Black-100);
-        font-size: 10px;
-        font-weight: bold;
-        margin-top: 4px;
-      }
+    &:not(:last-child) {
+      border-right: 1px solid var(--ui-2);
     }
+  }
 
-    .latest-priority-fee-field {
-      width: 40%;
-    }
+  &__field-data {
+    color: var(--ui-4);
```

Comment:
`ui-4` is now `var(--color-text-alternative)`

Reply:
Ah thanks — fixed in da86618c3d32853a418fe63688edffe99e4f8481.